pmultiqc/modules/quantms/quantms.py

```diff
@@ -643,53 +638,15 @@ def draw_peak_intensity_distribution(self):
             'title': 'Peak Intensity Distribution',
             # 'xlab': 'Peak Intensity'
         }
-        cats = OrderedDict()
-        cats['0-10'] = {
-            'name': '0-10',
-            'description': 'Peak intensity is between 0 and 10'
-        }
-        cats['10-100'] = {
-            'name': '10-100',
-            'description': 'Peak intensity is between 10 and 100'
-        }
-        cats['100-300'] = {
-            'name': '100-300',
-            'description': 'Peak intensity is between 100 and 300'
-        }
-        cats['300-500'] = {
-            'name': '300-500',
-            'description': 'Peak intensity is between 300 and 500'
-        }
-        cats['500-700'] = {
-            'name': '500-700',
-            'description': 'Peak intensity is between 500 and 700'
-        }
-        cats['700-900'] = {
-            'name': '700-900',
-            'description': 'Peak intensity is between 700 and 900'
-        }
-        cats['900-1000'] = {
-            'name': '900-1000',
-            'description': 'Peak intensity is between 900 and 1000'
-        }
-        cats['1000-3000'] = {
-            'name': '1000-3000',
-            'description': 'Peak intensity is between 1000 and 3000'
-        }
-        cats['3000-6000'] = {
-            'name': '3000-6000',
-            'description': 'Peak intensity is between 3000 and 6000'
-        }
-        cats['6000-10000'] = {
-            'name': '6000-10000',
-            'description': 'Peak intensity is between 6000 and 10000'
-        }
-        cats['>10000'] = {
-            'name': '>10000',
-            'description': 'peak intensity > 10000'
-        }
+        cats = self.mzml_spectra_plot.dict['cats']
+        for i in cats.keys():
+            cats[i].update({
+                'description': 'Peak intensity ' + 
```

Comment:
This should be done in Histogram.to_dict for all ranges. "Peak intensity" should be set in the constructor of Histogram to tell what the histogram will hold:
`self.peak_int_distr = Histogram("Peak intensity", breaks = [100,200,300])`